javascript/javascript3/week1/weatherApp/weather.js

```diff
@@ -0,0 +1,144 @@
+//Assigning api key to the variable
+const apiKey = '2a984eff06dbace9308d2bcf60b5e917';
+let weatherApi;
+
+//function that gets the html elements by id
+function getHtmlMarkup(idName) {
+    return document.getElementById(idName);
+}
+
```

Comment:
I would prefer to declare all HTML elements as global on top instead of using function-getHtmlMarkup